src/AppState.ts

```diff
@@ -163,6 +163,10 @@ export function useAppState(): {
   laterToday?: Catchable[];
   later?: Catchable[];
   alreadyCaught?: Catchable[];
+  filter: {
```

Comment:
This makes the most sense to be part of our main reducer and there isn't too much reason for it to be outside. The only reason that I have the current time outside is so that it isn't exposed to the rest of the app, but honestly it could be part of the `useReducer` state.